ucats/baselines.py

```diff
@@ -2,52 +2,40 @@
 baselines -- routines to estimate baseline fluorescence in 1D or TXY data
 """
 
-
 import pickle
-
 from functools import partial
 
-import numpy as np
-from numpy import linalg
-from numpy.linalg import norm, lstsq, svd, eig
-
-from scipy import ndimage as ndi
-
-from numba import jit
-
-
 import matplotlib.pyplot as plt
-
-
-from imfun import fseq
-from imfun.filt.dctsplines import l2spline,l1spline
-from imfun import components
-from imfun.multiscale import mvm
+import numpy as np
+from imfun import components, core
 from imfun.components.pca import PCA_frames
+from imfun.core import extrema
+from imfun.filt.dctsplines import l2spline, l1spline
+from imfun.multiscale import mvm
+from numba import jit
+from numpy.linalg import norm
+from scipy import ndimage as ndi
+from sklearn import decomposition as skd
```

Comment:
I would rather import all external/big modules first and my own ones last. Just a matter of taste